tsfile/src/test/java/org/apache/iotdb/tsfile/read/common/PathTest.java

```diff
@@ -41,7 +41,9 @@ public void construct() throws Exception {
     testPath(path, "", "", "");
     // with quote;
     path = new Path("root.d1.r1.\"x1.x2.x3\"");
-    testPath(path, "root.d1.r1", "x1.x2.x3", "root.d1.r1.\"x1.x2.x3\"");
+    testPath(path, "root.d1.r1", "\"x1.x2.x3\"", "root.d1.r1.\"x1.x2.x3\"");
```

Comment:
Also add some test about " in 'xxx' or ' in "xxx". It will make it more robust. Thanks~

Reply:
a path only supports a double quotes.